files/default/plugins/repolist.rb

```diff
@@ -0,0 +1,27 @@
+provides 'ohai_gecos'
+
+if ohai_gecos.nil?
+  ohai_gecos Mash.new
+end
+
+repofiles = []
+
+begin
+
```

Comment:
Buenas, Alfonso:

Me guío por lo que dice en esta O.T:
https://teo.i-administracion.junta-andalucia.es/issues/13387

Sólo he tenido en cuenta el directorio sources.list.d como se describe en
la O.T

Si se quiere que se contemple el sources.list, confírmamelo, que el cambio
es mínimo.

Saludos.

2016-08-01 10:06 GMT+02:00 Alfonso E.M. <notifications@github.com>:

> In files/default/plugins/repolist.rb
> <https://github.com/gecos-team/gecos-workstation-ohai-cookbook/pull/6#discussion_r72937778>
> :
>
> > @@ -0,0 +1,27 @@
> > +provides 'ohai_gecos'
> > +
> > +if ohai_gecos.nil?
> > +  ohai_gecos Mash.new
> > +end
> > +
> > +repofiles = []
> > +
> > +begin
> > +
>
> Main /etc/sources.list is not parsed
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gecos-team/gecos-workstation-ohai-cookbook/pull/6/files/193efbfd455437ea7f61933e4e134706b3a7f382#r72937778>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATHDTK3IX9p1CZ9jpKpmU6swWGAixbj6ks5qbajvgaJpZM4JX-jV>
> .
>